hyperledger-archives/sawtooth-docs-archive
# First steps for website refresh

To test:

`docker-compose up --build`
(You will see some warnings caused by files that haven't been updated for the new theme, this is expected.)

Open http://localhost:4000 in a browser.
